spynnaker/pyNN/extra_algorithms/splitter_components/splitter_abstract_pop_vertex_fixed.py

```diff
@@ -265,113 +254,51 @@ def __get_local_only_constant_sdram(self, n_atoms):
-            return self.__all_syn_block_sz[vertex_slice]
-        all_syn_block_sz = self._governed_app_vertex.get_synapses_size(
-            vertex_slice.n_atoms)
-        self.__all_syn_block_sz[vertex_slice] = all_syn_block_sz
-        return all_syn_block_sz
-
-    def __structural_size(self, vertex_slice):
-        """ Work out how much SDRAM is needed by the structural plasticity data
-
-        :param ~pacman.model.graphs.common.Slice vertex_slice:
-            The slice of neurons to get the size of
-        :rtype: int
-        """
-        if vertex_slice in self.__structural_sz:
-            return self.__structural_sz[vertex_slice]
-        structural_sz = self._governed_app_vertex.get_structural_dynamics_size(
-            vertex_slice.n_atoms)
-        self.__structural_sz[vertex_slice] = structural_sz
-        return structural_sz
-
-    def __synapse_expander_size(self):
-        """ Work out how much SDRAM is needed for the synapse expander
-
-        :rtype: int
-        """
-        if self.__synapse_expander_sz is None:
-            self.__synapse_expander_sz = \
-                self._governed_app_vertex.get_synapse_expander_size()
-        return self.__synapse_expander_sz
-
-    def __bitfield_size(self):
-        """ Work out how much SDRAM is needed by the bit fields
-
-        :rtype: ~pacman.model.resources.MultiRegionSDRAM
-        """
-        if self.__bitfield_sz is None:
-            sdram = MultiRegionSDRAM()
-            sdram.add_cost(
-                PopulationMachineVertex.SYNAPSE_REGIONS.bitfield_filter,
-                get_estimated_sdram_for_bit_field_region(
-                    self._governed_app_vertex.incoming_projections))
-            sdram.add_cost(
-                PopulationMachineVertex.SYNAPSE_REGIONS.bitfield_key_map,
-                get_estimated_sdram_for_key_region(
-                    self._governed_app_vertex.incoming_projections))
-            sdram.add_cost(
-                PopulationMachineVertex.SYNAPSE_REGIONS.bitfield_builder,
-                exact_sdram_for_bit_field_builder_region())
-            self.__bitfield_sz = sdram
-        return self.__bitfield_sz
-
     @overrides(AbstractSplitterCommon.reset_called)
     def reset_called(self):
         super(SplitterAbstractPopulationVertexFixed, self).reset_called()
-        self.__ring_buffer_shifts = None
-        self.__weight_scales = None
-        self.__all_syn_block_sz = dict()
-        self.__structural_sz = dict()
-        self.__next_index = 0
         self.__slices = None
-        self.__vertices = list()
 
     def __create_slices(self):
         """ Create slices if not already done
         """
         if self.__slices is not None:
             return
         self.__slices = get_multidimensional_slices(self._governed_app_vertex)
+
+    @property
+    def n_synapse_vertices(self):
+        """ Return the number of synapse vertices per neuron vertex
+
+        :rtype: int
+        """
+        return 1
```

Comment:
Why not zero?
There are no synapse vertices

Reply:
This appears to be no longer called so removed!  If it was to stay, it was probably 1 for compatibility purposes...